src/components/TimeseriesChart.js

```diff
@@ -734,15 +725,8 @@ class TimeseriesChartComponent extends Component {
           <Plot
             className="fullPlot"
             data={combinedEvents}
-            layout={tmpLayout}
+            layout={this.getLayout(this.state.wantedAxes, thresholds)}
```

Comment:
die wantedAxes had ik weggehaald, dit moet gewoon `this.getLayout(axes, thresholds)` zijn.